logging/log.go

```diff
@@ -0,0 +1,52 @@
+package logging
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+)
+
+// TODO: Should this be a pointer?
+// https://github.com/rs/zerolog/issues/1
+var Logger zerolog.Logger
+
+func init() {
+	// send all logs to stdout
+	Logger = zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr}).
+		Level(zerolog.InfoLevel).
+		With().Timestamp().Logger()
```

Comment:
I need to check if there are any other differences between this and the defaults from `log.Output(zerolog.ConsoleWriter{Out: os.Stderr})`.

Reply:
It looks the same as far as I can tell. 🤷‍♂️